### ISSUE TITLE
LuckPermsGroupInfo Sorting Rule [Question]

### ISSUE THREAD

**Issue by Joapple** [CodeCrafter47/BungeeTabListPlus#206]
Not sure if i open this at the right place, if not i'll go and move to LuckPerms, but for bungee and spigot we are using Luckperms. I want to list my Staff with both this sorting rule and alphabetically, no mather what i a try, they only get sorted alphabetically. 

BungeeTabListPlus v2.7.3
Luckpermsv3.0.13(bungee)

Thanks for your time!

**Comment by CodeCrafter47** [comment]
Use
```yaml
playerOrder: "luckPermsGroupInfo,alphabetically"
```

**Comment by Joapple** [comment]
I already tried it, but did it again, just to be sure, they only get sorted alphabetically, my staff aren't sorted by groups :/

**Comment by CodeCrafter47** [comment]
You need to set the weight options for all groups as that is what BumngeeTabListPlus uses to sort the players.

**Comment by Joapple** [comment]
Yeah did that already, all groups has their weight. It works for the placeholder from placeholderAPI but i can't use that as a rule.

**Comment by CodeCrafter47** [comment]
Maybe this helps: <https://www.spigotmc.org/threads/bungeetablistplus.10325/page-129#post-2301008>

**Comment by Joapple** [comment]
Tested luckperms v3.0.35 and still doesn't work, my groups have all weight, but doesn't get sorted by group...

**Comment by CodeCrafter47** [comment]
Can you try displaying the player's primary group in the tab list using the `${viewer luckperms_primary_group}` placeholder just to check whether BungeeTabListPlus correctly interacts with LuckPerms.

**Comment by Joapple** [comment]
Yeah that place holder works perfectly, i have a "Info" section which display the primary group without any weird error. That's why i have weight set already to have that section working. If you check after name, it display the group moderator, admin, builder. It should list groups by Owner, then admin, then builder, then moderator, but it's not doing it for some reason. 
![2017-03-17_11 36 21](https://cloud.githubusercontent.com/assets/22040491/24050683/ffb60028-0b05-11e7-88ae-75ca9b0ec92f.png)

**Comment by CodeCrafter47** [comment]
I have no idea why it's not working.

**Comment by BrainStone** [comment]
Any updates on this?

**Comment by CodeCrafter47** [comment]
See <https://github.com/CodeCrafter47/BungeeTabListPlus/issues/263#issuecomment-321603331>.
